lib/inventory_manipulation.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+# helper functions for manipulating and reading a bolt inventory file
+module InventoryManipulation
```

Comment:
Should it be namespaced so it ends up as `Litmus::InventoryManipulation`?

Reply:
not 100% sure. TBH. It would be more practical long term for this to live in the bolt gem, leaving it as a simple module would aid this. 
However the full name does make it clear where the functionality is coming from.

Reply:
namespaced it